docker-compose.override.yml

```diff
@@ -0,0 +1,5 @@
+version: "2.3"
+
+services:
+  addons-frontend:
+    build: ../.
```

Comment:
I like how simple this override is 👍 This will share most configuration with `addons-server` which is good.

Reply:
I will have to patch addons-server though, but that is no biggie.